data/blog/how-we-improved-performance-score-accuracy.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'How we improved Performance Score accuracy'
+date: '2024-01-17'
+tags: ['javascript', 'performance', 'web']
+draft: true
+summary: "We're making updates to how Performance Scores are calculated in the Web Vitals module, which will bring them closer to what your users experience."
+images: ['/images/how-we-improved-performance-score-accuracy/hero.jpg']
+layout: PostLayout
+canonicalUrl: performance-score-improvements
+authors: ['edwardgou']
+---
+
+Last year in November, we released the Web Vitals module for Sentry's Performance product. Aside from helping users monitor and improve their Web Vitals, this new module also introduced [Sentry Performance Scores](https://docs.sentry.io/product/performance/web-vitals/#performance-score) for all web browser applications. As a quick reminder, Performance Scores are used to condense multiple Web Vitals and their respective thresholds into an overall score from 0-100 that rates an app's perceived performance, based on real user data for Web Vitals.
+
+Today, we are making updates to how Performance Scores are calculated. These updates will improve the accuracy of these scores by bringing them closer to what users experience on your web apps. To understand how these changes will impact your Performance Scores, it's useful to understand how they used to be calculated. Here's a brief overview:
```

Comment:
This makes sense, but the tone is more like a product announcement than a technical post. This is why I was curious why this is an engineering blog post rather than the regular blog, since the style is more of a product announcement

Reply:
👍 noted. I agree, there's a pretty big product announcement element to this post. We're choosing this avenue because there's a lower barrier here, we want to get this message out asap, and there are some technical details that feel suitable here. @zoesyc 

I've updated the content in this post with more of the technical content relating to your feedback below, so hopefully this post doesn't feel out of place anymore.